src/extension.ts

```diff
@@ -127,13 +131,30 @@ export function activate(context: vscode.ExtensionContext) {
 			}
 		}
 	}
+
+	context.subscriptions.push(
+		vscode.debug.onDidStartDebugSession((session) => {
+			const config = session.configuration as LaunchConfiguration;
+			adapterDescriptorFactory.get_version(config).then((strVer) => {
+				if (strVer === null) return;
+				const version = adapterDescriptorFactory.vernum(strVer);
+				if (version >= 1008000) {
```

Comment:
What does `1008000` refer to here?

Reply:
Could this be declared as a constant with comments? It's usually a good practice for values like this so readers can understand it right away.